src/components/organisms/videoList/VideoList.tsx

```diff
@@ -2,18 +2,38 @@ import { Grid } from '@mui/material'
 import { IVideoInfo } from '../../../common/interfaces/IVideoInfo'
 import { PageTitle } from '../../atoms/pageTitle/PageTitle'
 import { VideoCard } from '../../molecules/videoCard/VideoCard'
+import { ShowMoreButton } from '../../atoms/buttons/ShowMoreButton'
+import { HireMeButton } from '../../atoms/buttons/HireMeButton'
+import { useState } from 'react'
+import { useLocation } from 'react-router-dom'
 
 type VideoListProps = {
   videos: IVideoInfo[]
   pageType: 'genre' | 'role' | 'recent'
 }
 
+const videosPerPage = 3
+
 export function VideoList({ videos, pageType }: VideoListProps) {
-  const videoComponents = videos.map(video => (
+  const location = useLocation()
+  const isHomePage = location.pathname === '/'
```

Comment:
❓ If the path is: `localhost:8080/#contact`, what does `location.pathname` return? 

Does it return `/` or `/#contact`?

Reply:
You were 100% right. 
`Console.log(location.pathname)` is returning `/` when it should be returning `/#contact` here.